packages/api-gateway-service/src/stitch-schema.ts

```diff
@@ -1,21 +1,19 @@
 import fs from 'fs';
 import { stitchSchemas } from 'graphql-tools';
-import { getRemoteSchema } from './helpers';
+import remoteSchema from './remote-schema';
 
 export const microservices = loadConfig();
 
-export async function stitchedSchemas () {
-  const schemas = await Promise
-    .all( microservices
-      .map( service => {
-        console.log( `Loading microservice: ${ service.name }` );
-        return getRemoteSchema( service )
-          .catch( err => {
-            console.error( '[Error]:', err.message );
-            console.log( '... Gateway will continue without:', service.name );
-            return '';
-          } );
-      } ) );
+export async function stitchedSchemas() {
+  const schemas = await Promise.all( microservices.map( service => {
+    console.log( `Loading microservice: ${ service.name }` );
+    return remoteSchema( service )
+      .catch( err => {
+        console.error( '[Error]:', err.message );
+        console.log( '... Gateway will continnue without:', service.name );
```

Comment:
```suggestion
        console.log( '... Gateway running without :', service.name );
```

Reply:
Changed it `... Skipping: service.name` instead, as there's another check below this which ensures that at least 1 of the services are connected before it can start the API Gateway. So saying "Gateway running" is not entirely accurate.